packages/jellyfish-transaction-builder/src/txn/txn_builder_oracles.ts

```diff
@@ -0,0 +1,27 @@
+import { AppointOracle } from '@defichain/jellyfish-transaction/dist/script/defi/dftx_oracles'
+import { OP_CODES, Script, TransactionSegWit } from '@defichain/jellyfish-transaction'
+import { P2WPKHTxnBuilder } from './txn_builder'
+import { TxnBuilderError, TxnBuilderErrorType } from './txn_builder_error'
+
+export class TxnBuilderOracles extends P2WPKHTxnBuilder {
+  /**
+   * Requires UTXO in the same amount + fees to create a transaction.
```

Comment:
```suggestion
```

Reply:
This description seems invalid

Reply:
oh yes I fixed in #307, will bring the change over